src/cgame/cg_servercmds.cpp

```diff
@@ -415,6 +415,7 @@ static void CG_Menu( int menuType, int arg )
 	switch ( menuType )
 	{
 		case MN_WELCOME:
+			Rocket_DocumentAction( "changelog", "show" );
```

Comment:
Let's not hardcode these things into the game and have them be configurable.

Referring to "changelog"

Reply:
I don't really think we want everything to be configurable.
What would be the benefits to change this?
It would only make support harder when someone changes it, and comes saying that their stuff do not work because they typo'ed somewhere.